docs/plugins/pluggable_rule_engine.md

```diff
@@ -92,48 +90,55 @@ Consider the following `server_config.json` with the accompanying `example.re`,
 }
 ```
 
+#### core.py
 ```python
-# core.py
+RULE_ENGINE_CONTINUE = 5000000
+
 def pep_resource_close_pre(rule_args, callback, rei):
-    callback.writeLine("serverLog","XXXX - core.py")
+    callback.writeLine("serverLog", "XXXX - core.py")
+
+    # Instruct the REPF to let the next rule engine plugin handle this PEP.
+    # If we remove the line below, the PEP in core.re will not execute. If an
+    # error occurs before reaching the line below, processing stops and the
+    # error is returned to the client. No other rule engines will be allowed
+    # to process the PEP.
+    return RULE_ENGINE_CONTINUE
```

Comment:
I've added more words to the code comment to explain what happens if an error occurs before reaching the continuation code.

Thoughts?